docs/website-docs/record-n-upload.md

```diff
@@ -0,0 +1,125 @@
+Android
+---------
+
+## Record
+
+### How to start a trace recording
+The 'Record trace' button should be available in your quick settings. If it is not, go back to the section on [enabling trace recording](device-set-up.md#3-make-sure-tracing-is-enabled) and ensure you enabled __Show Quick Settings tile__.
```

Comment:
Could add "tile" to the end of the first sentence  to be clear and include an image of the quick settings tile